src/agent/ICheqd.ts

```diff
@@ -6739,6 +6826,11 @@ export class Cheqd implements IAgentPlugin {
 			);
 		}
 
+		// validate dkgOptions
+		if (!options?.topArgs?.dkgOptions || !options.topArgs?.dkgOptions?.capacityDelegationAuthSignature) {
```

Comment:
@Eengineer1 should I rename `capacityDelegationAuthSignature` to something simpler

Reply:
No need as it captures the necessary semantics, no extra information.